proposals/lip-0057.md

```diff
@@ -1519,10 +1533,10 @@ def isNameAvailable(name: str) -> bool:
         return True
 ```
 
-#### getPoSTokenID
+#### getPosTokenID
 
 ```python
-def getPoSTokenID() -> TokenID:
+def getPosTokenID() -> TokenID:
```

Comment:
:( this will be recurring change everywhere? is this important?

Reply:
also, `PoS` seems correct?

Reply:
@ManuGowda No, this is not that important.

Reply:
@janhack Do we need to revert the it? or we can keep it like this in the LIP?

Reply:
As discussed, I reverted the name of the endpoint to the previous version.